Mage.Sets/src/mage/sets/Warhammer40000.java

```diff
@@ -150,7 +150,7 @@ private Warhammer40000() {
         cards.add(new SetCardInfo("Inspiring Call", 217, Rarity.UNCOMMON, mage.cards.i.InspiringCall.class));
         cards.add(new SetCardInfo("Island", 307, Rarity.LAND, mage.cards.basiclands.Island.class, NON_FULL_USE_VARIOUS));
         cards.add(new SetCardInfo("Keeper of Secrets", 78, Rarity.RARE, mage.cards.k.KeeperOfSecrets.class));
-        cards.add(new SetCardInfo("Khârn the Betrayer", 79, Rarity.RARE, mage.cards.k.KharnTheBetrayer.class));
+        cards.add(new SetCardInfo("Kharn the Betrayer", 79, Rarity.RARE, mage.cards.k.KharnTheBetrayer.class));
```

Comment:
I think we want to keep the diacritics in, but clean them to their simple letter in VerifyCardDataTest (or before generating the mtgjson file we compare the cards against.)
See #10831

Reply:
Looks like LTR set doesn't include diacritics here though

Reply:
I tried checking for diacritics in other sets like LTR, and I didn't see them there, so I assumed they weren't put in

Reply:
So there should be no diacritics at all anywhere in the end? Works for me too.